examples/v3/e2e/test/provider.spec.js

```diff
@@ -4,7 +4,7 @@ const chaiAsPromised = require('chai-as-promised');
 chai.use(chaiAsPromised);
 const { server, importData, animalRepository } = require('../provider.js');
 const path = require('path');
-const LOG_LEVEL = process.env.LOG_LEVEL || 'TRACE';
+const LOG_LEVEL = process.env.LOG_LEVEL || 'INFO';
```

Comment:
it might be nicer to set info by default and set to trace on a workflow input in CI, so we get trace logs when we need them but the output is sane otherwise.

Reply:
agree, thanks.